src/remove-dulplicates/index.js

```diff
@@ -1,3 +1,28 @@
-function removeDuplicates(obj) {}
+function removeDuplicates(obj) {
+  let myKeys = Object.keys(obj);
+  let keysHierarchy = myKeys.sort((a, b) => a - b);
+  let compareArr = [];
+  let myReturnObj = {};
+  keysHierarchy.forEach((element) => {
+    myReturnObj[element] = obj[element];
+  });
+  for (let i = keysHierarchy.length - 1; i >= 0; i -= 1) {
+    let objArr = myReturnObj[keysHierarchy[i]];
+
+    for (let j = 0; j < objArr.length; j += 1) {
+
+      let objArrItem = myReturnObj[keysHierarchy[i]][j];
+
+      if (!compareArr.includes(objArrItem)) {
+        compareArr.push(objArrItem);
+      } else if (compareArr.includes(objArrItem)) {
+        objArr.splice(j, 1);
+        //console.log(myReturnObj[keysHierarchy[i]])
+        j -= 1;
```

Comment:
This is counter-intuitive but works. The loop is incrementing `j` but you're taking it back, makes it difficult to understand what is going on.